docs/api/app.md

```diff
@@ -1307,6 +1307,76 @@ Sets the application's [dock menu][dock-menu].
 
 Sets the `image` associated with this dock icon.
 
+### `app.sysInfo.numberOfProcessors()`
+
+Returns `Number` - The number of logical processors/cores on the current machine.
+
+### `app.sysInfo.amountOfPhysicalMemory()`
+
+Returns `Number` - The number of bytes of physical memory on the current machine.
+
+### `app.sysInfo.amountOfAvailablePhysicalMemory()`
+
+Returns `Number` - The number of bytes of current available physical memory on the machine.
+
+### `app.sysInfo.amountOfVirtualMemory()`
+
+Returns `Number` - The number of bytes of virtual memory of this process.
+
+**NOTE:** A return value of zero means that there is no limit on the available virtual memory.
+
+### `app.sysInfo.amountOfFreeDiskSpace(path)`
+
+* `path` String
+
+Returns `Number` - The available disk space in bytes on the volume containing `path`.
+
+**NOTE:** Returns -1 on failure.
+
+### `app.sysInfo.amountOfTotalDiskSpace(path)`
+
+* `path` String
+
+Returns `Number` - The total disk space in bytes on the volume containing `path`.
+
+**NOTE:** Returns -1 on failure.
+
+### `app.sysInfo.uptime()`
+
+Returns `Number` - The system uptime in seconds.
+
+### `app.sysInfo.getHardwareInfo()`
+
+Returns `Promise<HardwareInfo>` - Resolves with a [HardwareInfo](structures/hardware-info.md)
+
+### `app.sysInfo.operatingSystemName()`
+
+Returns `String` - The name of the host operating system.
```

Comment:
Can we specify some values? `name` means so many different things. Like, is it `win32`, `Windows`, `Windows 10`, or `Windows 10N`?